src/kernels/jupyter/jupyterConnection.ts

```diff
@@ -65,7 +66,7 @@ export class JupyterConnection implements IExtensionSyncActivationService {
     private async getUriFromServerId(serverId: string) {
         // Since there's one server per session, don't use a resource to figure out these settings
         const savedList = await this.serverUriStorage.getSavedUriList();
-        return savedList.find((item) => computeUriHash(item.uri) === serverId)?.uri;
+        return savedList.find((item) => computeServerId(item.uri) === serverId)?.uri;
```

Comment:
Renamed `computeUriHash` to be `computeServerId`, the implementation should not define the method name, its a server id, hence the name should reflect that.